app/src/main/java/com/droidhats/campuscompass/MapsActivity.kt

```diff
@@ -422,19 +461,55 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback,  GoogleMap.OnMarke
         val lib_Polygon: Polygon = map.addPolygon(lib_PolygonOptions)
         lib_Polygon.tag = getString( R.string.WebsterLibrary_Building_Name )
 
-    //FG Building
+        //FG Building
         val fg_PolygonOptions = PolygonOptions()
             .clickable(true)
             .add(
-               LatLng(45.493822, -73.579069),
-               LatLng(45.493619, -73.578735),
-               LatLng(45.494457, -73.577627),
-               LatLng(45.494687, -73.578045),
-               LatLng(45.494363, -73.578439)
-        )
+                LatLng(45.493822, -73.579069),
+                LatLng(45.493619, -73.578735),
+                LatLng(45.494457, -73.577627),
+                LatLng(45.494687, -73.578045),
+                LatLng(45.494363, -73.578439)
+            )
         val fg_Polygon: Polygon = map.addPolygon(fg_PolygonOptions)
         fg_Polygon.tag = getString( R.string.FG_Building_Name )
 
     }
 
+    private fun initBottomSheetBehavior()
+    {
+        bottomSheetBehavior = BottomSheetBehavior.from(bottom_sheet)
+
+        bottomSheetBehavior.setBottomSheetCallback(object : BottomSheetBehavior.BottomSheetCallback() {
+
+            override fun onStateChanged(bottomSheet: View, newState: Int)
+            {
+                // React to state change
+                when (newState)
+                {
+                    BottomSheetBehavior.STATE_HIDDEN -> {
+                    }
+                    BottomSheetBehavior.STATE_EXPANDED -> {
+                    }
+                    BottomSheetBehavior.STATE_COLLAPSED -> {
+                    }
+                    BottomSheetBehavior.STATE_DRAGGING -> {
+                    }
+                    BottomSheetBehavior.STATE_SETTLING -> {
+                    }
+                    BottomSheetBehavior.STATE_HALF_EXPANDED -> {
+                    }
```

Comment:
Is it necessary to really have all these empty lambda expressions?

Reply:
No it is not

Reply:
We still need this function as it implements an abstract method, so I commented it out, explained a little as we might use it later